data/coalition/coalition missions.txt

```diff
@@ -1866,3 +1866,469 @@ mission "Saryd University Lecture"
+			`	"Cities buried whole by lava? Tuned their scanners to look for such odd ruins, no expedition must have..." she stops and thinks for a few minutes, working on the pylon's command panel. "Tell the story again, could you, Captain <last>? Formally record the findings here, I would like." You spend the next few hours retelling what you saw, and helping her search what information humans have gotten on the Sheragi and their ruins since Foster published his initial findings.`
+			`	Once she's satisfied, you two look to your map, to match coordinates with the next stop Mimodum made.`
+				goto end
+			label notknow
+			`	She frowns slightly. "Figured as much, I had. Well, finding out more about them, the point of this expedition wasn't. A nice bonus, it would've been, but prepare to look for the next pylon, we should."`
+			label end
+			`	You open your map, and the coordinates point to "Summit" being <planet>, a small moon close to the entrance to The Deep. Sedlitaris returns to her bunk, and you set course to <system>.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 7"
+	landing
+	name "Curious Red Cloud"
+	description "Look for the system near Algieba where Mimodum, the expedition's leader, jumped to investigate the cosmological phenomenon in the system."
+	passengers 1
+	to offer
+		has "Coalition: Expeditions Past 6: done"
+	source "Watcher"
+	on offer
+		conversation
+			`You fly around the moon as Sedlitaris nails down the coordinates of the pylon, until you land close to one of the forests. "Deeper inside, the pylon must be. Uninhabited by humans this world is, correct?" You confirm it, and she prepares to leave. "No giant dragons to worry about here, there are, so try to bring back this pylon with us, I will."`
+			`	You two leave your ship and enter the forest, taking turns carrying the pylon you brought, which is easy enough to carry due to the lower gravity. Occasionally Sedlitaris stops and has the pylon put down so that she can determine where to go next. Eventually, you notice the trees start getting smaller as you move on, until you reach a wide, odd clearing where the ground is devoid of any of the local flora. Sedlitaris spots the other pylon on the opposite side, half stuck in the ground, and moves to pull it out. As you follow her to the machine, you notice deep, perfectly straight holes in the ground, forming lines that sometimes cross the entire clearing and continue well past it. Soil and dust have filled most of them up, but in certain gaps you can look down to cracks that must be dozens of meters deep.`
```

Comment:
```suggestion
			`	You two leave your ship and enter the forest, taking turns carrying the pylon you brought, which is easy enough to do since the gravity is so much lower here. Occasionally Sedlitaris stops and has the pylon put down so that she can determine where to go next. Eventually, you notice the trees start getting smaller as you move on, until you reach a wide, odd clearing where the ground is devoid of any of the local flora. Sedlitaris spots the other pylon on the opposite side, half stuck in the ground, and moves to pull it out. As you follow her to the machine, you notice deep, perfectly straight holes in the ground, forming lines that sometimes cross the entire clearing and continue well past it. Soil and dust have filled most of them up, but in certain gaps you can look down to cracks that must be dozens of meters deep.`
```